lib/web_ui/dev/build.dart

```diff
@@ -73,39 +67,19 @@ class BuildCommand extends Command<bool> with ArgUtils<bool> {
 /// Not safe to interrupt as it may leave the `out/` directory in a corrupted
 /// state. GN is pretty quick though, so it's OK to not support interruption.
 class GnPipelineStep extends ProcessStep {
-  GnPipelineStep({this.target = 'engine'})
-      : assert(target == 'engine' || target == 'canvaskit');
-
   @override
   String get description => 'gn';
 
   @override
   bool get isSafeToInterrupt => false;
 
-  /// The target to build with gn.
-  ///
-  /// Acceptable values: engine, canvaskit
-  final String target;
-
   @override
   Future<ProcessManager> createProcess() {
-    print('Running gn for $target...');
-    final List<String> gnArgs = <String>[];
-    if (target == 'engine') {
-      gnArgs.addAll(<String>[
-        '--unopt',
-        if (Platform.isMacOS) '--xcode-symlinks',
-        '--full-dart-sdk',
-        if (environment.isMacosArm) '--mac-cpu=arm64',
-      ]);
-    } else if (target == 'canvaskit') {
-      gnArgs.addAll(<String>[
-        '--wasm',
-        '--runtime-mode=release',
-      ]);
-    } else {
-      throw StateError('Target was not engine or canvaskit: $target');
-    }
+    print('Running gn...');
+    const List<String> gnArgs = <String>[
+      '--wasm',
```

Comment:
Now that this target is going to be used for all of flutter web (not just for wasm), does it make sense to change the name to `flutter_web` or similar?

Reply:
I think there was some extensive discussion about this with the rest of the engine team, I'd like to ask @hterkelsen what their concerns were and why they landed on this. In any case, the `--wasm` can't be immediately deleted because the recipes use it, but if we are going to migrate to some different flag I should at least add it as an option here so the recipes can switch to it.

Reply:
My concern with using `--wasm` to mean "build the Flutter Web SDK" is that the Flutter Web SDK bits aren't WASM (for now, anyway). The fact that we can invoke kernel compiler when using the WASM toolchain seems to just be a lucky coincidence. I think we should have a `--web` flag in GN which sets the default CPU to WASM and builds CanvasKit and the Flutter Web SDK.

Reply:
That makes sense. I think I get what you're going for. Let me make some changes and see what I can do.